src/QA/QA.Web/Views/Dashboard/Index.cshtml

```diff
@@ -119,6 +119,10 @@
                     <td class="govuk-table__cell">
                         <span asp-show="@vacancy.ShowAssignmentInfoCaption">@assignmentInfoCaption<br /></span>
                         <a asp-show="@vacancy.CanShowReviewLink" asp-route="@RouteNames.Vacancy_Review_Get" asp-route-reviewId="@vacancy.ReviewId" class="govuk-link">Review</a>
+                        @if (vacancy.IsVacancyDeleted)
+                        {
+                            <p>This vacancy has been deleted</p>
```

Comment:
Can you use the tag helper instead of the `if` statement `<p asp-show="@vacancy.IsVacancyDeleted">`